data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,391 @@ mission "Remnant: Face to Maw 2B"
+				goto deliverdata
+			label foodvolume
+			branch capitallicense
+				"license: Remnant Capital" = 1
+			`	He looks at you appraisingly. "You are perceptive, but there are some things you cannot learn yet. Ask us again when you have earned more of our trust."`
+				goto deliverdata
+			label capitallicense
+			apply "remnant symbionts info" += 1
+			`	He looks at you appraisingly "You are perceptive, and have earned our trust." He leans forward and holds up his hands. He pokes at the leather fingerless gloves he is wearing, and you note that they actually extend up under his sleeves. Looking closer, you note that the leather does not actually look like any leather you have seen - it has fine scales and rough ridges on it. What stands out the most, however, is that it appears to pulse faintly.`
+			`	"The truth is, when we eat, we are eating for two. You will have to ask someone else how they came to be or how it works, but suffice to say that we have developed a symbiotic relationship with a variety of creatures we found here. These ones." He holds up his hands. "Have a knack for precision and durability. With them, I am more precise than I would otherwise be, and much tougher. In exchange, they feed on us and require regular care and attention. Normally, this causes us no hardship; but if we do not eat enough we will consume our reserves far faster than we normally would."`
+			choice
+				`	"That is quite the revelation."`
+				`	"Thanks for the explanation."`
+				`	"Is that why some of the food tastes terrible?"`
+					goto terribleflavor
+				`	"Sounds creepy."`
+					goto creepy
+			`	The outfitter nods. "From what I have heard, humanity does not even seem to have much in the way of exoskeletons or cybernetics. Having living organisms we wear would certainly seem to be the realm of science-fiction." He contemplates his symbiont for a few seconds as he absent-mindedly eats more of his meal. "But education aside, there is work to do."`
+				goto deliverdata
+			label spices
+			`	He shakes his head. "I will not discourage you doing so - and I am sure some might appreciate them - but most of us like our food the way it is." He chuckles, then continues. "But there is still work to do."`
+				goto deliverdata
+			label terribleflavor
+			`	"I suppose it might. Our symbionts have different dietary requirements, and such unusual compounds in our food inevitably change the flavor. We are all long since used to it, but it would definitely be different for an outsider." He seems amused by this. "But enough of food - there is work to do!"`
+				goto deliverdata
+			label creepy
+			`	"Creepy? I do not understand how something so useful and beneficial could be 'creepy,' but I suppose some might equate our symbionts to being useful leeches. To each their own, I suppose." He shrugs, then continues. "But enough. We still have work to do."`
```

Comment:
Not this specifically, but about the entire above conversation: nice work making this section here branching. Having him specifically respond to questions and statements you say seems more natural. The only thing I would warn is if symbionts are going to play a role in future plot points, ensure they are either explained again then (because I believe the player can go through this without ever learning about symbionts) or alter it so the player has to learn about symbionts no matter what conversation options they pick.

Reply:
Thank you. 

This is intended to be something of a "sneak peek" for the player. It will be a plot point in the future, and before then they will have more direct opportunities to learn about it.

Reply:
Good to hear (resolved).